cmd/pocket/main.go

```diff
@@ -142,6 +149,32 @@ func commandArchive(arguments map[string]interface{}, client *api.Client) {
 	}
 }
 
+func commandAdd(arguments map[string]interface{}, client *api.Client) {
+	options := &api.AddOption{}
+
+	url, ok := arguments["<url>"].(string)
+	if !ok {
+		panic("Wrong arguments")
+	}
+
+	options.Url = url
+
+	if title, ok := arguments["--title"].(string); ok {
+		options.Title = title
+	}
+
+	if tags, ok := arguments["--tags"].(string); ok {
+		options.Tags = tags
+	}
+
+	err := client.Add(options)
+	if err != nil {
+		panic(err)
+	}
+
+	fmt.Println("Successfully added.")
```

Comment:
I'd like have this part:

- if `err != nil`, print it out to stderr and exit with nonzero
- If `err == nil`, print nothing and exit with zero

If you don't mind, I myself can modify this part and merge this PR :smile: